geniza/corpus/metadata_export.py

```diff
@@ -112,6 +136,10 @@ def get_export_data_dict(self, doc):
         outd["type"] = doc.doctype
         outd["tags"] = doc.all_tags()
         outd["description"] = doc.description
+        # include short display version of scholarship records;
+        # need to use set since some sources have duplicate footnotes
+        # in order to keep track of multiple links / PDFs
+        outd["scholarship_records"] = set([fn.display() for fn in doc.footnotes.all()])
```

Comment:
I like using the set comprehension syntax `{fn.display() for fn in doc.footnotes.all()}` but it's just style

Reply:
Ah! I don't think I knew you could do that. That does look nicer. (Although I confess that I do get a little bit of visual confusion between that kind of set notation and dicts sometimes)